.ci.yaml

```diff
@@ -39,6 +39,7 @@ targets:
     recipe: plugins/plugins
     timeout: 30
     properties:
+      add_recipes_cq: "true"
```

Comment:
We want the string true, not the value `true`?

Reply:
Thanks for pointing this out! I'll add it to the docs.

This is a limitation of yaml files backed by proto (we define properties to be Map<String, String> as there's no Map<String, dynamic> option (granted we could do a JsonString of sorts)